src/platforms/hosted/cli.c~HEAD

```diff
@@ -0,0 +1,654 @@
+/*
```

Comment:
This looks very very wrong - think you didn't mean to rename cli.c to `cli.c~HEAD` - This looks like an unresolved merge conflict as ~master is the version in the repo already - we suggest doing `git mv cli.c~HEAD cli.c`, `git rm cli.c~master` and then `git commit --amend` followed by a force-push